### ISSUE TITLE
Ender 5 using the new creality 4.2.7 card and the new V2 touch screen (maybe the CR6 screen) 

### ISSUE THREAD

**Issue by axpaul** [MarlinFirmware/Configurations#563]
### Description 

Recently, I bought this kit including the new touch screen normally dedicated to the ender 3. Having bought the creality 4.2.7 card, I thought it might work even if it's not the same structure, but I couldn't configure it on merlin. I think this screen is normally intended for cr6 lien : https://www.amazon.fr/Creality-pouces-tactile-niveau-imprimante/dp/B08RN5F5RQ/ref=sr_1_37?__mk_fr_FR=%C3%85M%C3%85%C5%BD%C3%95%C3%91&crid=182P08CF9CY10&dchild=1&keywords=ender+3+upgrade&qid=1630682653&sprefix=ender+3+up%2Caps%2C164&sr=8-37

![image](https://user-images.githubusercontent.com/38831329/132031162-bee372ea-fb0b-40f7-b7b0-7a0fe49cf8d2.png)

### Configuration Details

Is it possible to have a configuration to run this screen with the settings of an ender 5 with a creality 4.2.7 board? 

* Manufacturer: Creality
* Model: ender 5
* Motherboard: Creality 4.2.7

#### Additional Information

I thank you for your work and do not hesitate to tell me if it is impossible or not

**Comment by ellensp** [comment]
That TFT is not supported in Marlin currently, but will once MarlinFirmware/Marlin#19958 is merged.

Duplicate of https://github.com/MarlinFirmware/Configurations/issues/554
